network/load-balancer/reference-content/configuring-frontends.mdx

```diff
@@ -57,7 +57,7 @@ You can only add an SSL certificate to a frontend after creation of the Load Bal
 
 You can add an SSL/TLS certificate to your frontend to allow clients to securely connect to the Load Balancer via an encrypted connection. This is necessary for [SSL bridging and SSL offloading configurations](/network/load-balancer/reference-content/ssl-bridging-offloading-passthrough/).
 
-When creating a certificate you can generate a Let's Encrypt certificate directly from the console, or import an existing certificate. This could be from a third-party Certificate Authority (CA) other than Let's Encrypt, or be a self-signed certificate.
+When creating a certificate you can generate a Let's Encrypt certificate directly from the console, or import an existing certificate. This could be from a third-party Certificate Authority (CA), or be a self-signed certificate.
```

Comment:
```suggestion
When creating a certificate, you can generate a Let's Encrypt certificate directly from the console or import an existing one. This could be a self-signed certificate or a third-party Certificate Authority (CA).
```